app/views/articles/index.html.erb

```diff
@@ -30,7 +30,7 @@
       data-params="<%= params.to_json(only: %i[tag username q]) %>" data-which="<%= @list_of %>"
       data-tag=""
       data-feed="<%= params[:timeframe] || "base-feed" %>"
-      data-articles-since="<%= Timeframer.new(params[:timeframe]).datetime.to_i %>">
+      data-articles-since="<%= Timeframer.new(params[:timeframe]).datetime&.iso8601 %>">
```

Comment:
where does this get parsed? I was trying to find it in the code, to make sure that it wouldn't break with empty string but I can't find it in the JS

Reply:
https://github.com/thepracticaldev/dev.to/pull/6847/files#diff-0869f8ed3a2fce756f2dc6a68000ed74R199

If it is a blank value we will remove it from the params in the controller.